src/cli2/common.js

```diff
@@ -59,3 +60,53 @@ export function pluginDirectoryContext(
     },
   };
 }
+
+/**
+ * Load and parse a JSON file from disk.
+ *
+ * If no file is present at that location, then an error is thrown.
```

Comment:
“If the file cannot be read, …”. (Also throws on EACCES, e.g.).

Likewise below.